### PR TITLE
fix quizservice

### LINKED ISSUE #281: Fix quizService
addQuestion method does not work

Also, the endpoint is plain wrong

### PR DESCRIPTION
**Pull request by Guekka**
Fixes #281

### CODE REVIEW

File: apps/front-end/back-office/src/app/components/quiz-edition/quiz-edition.component.ts [hunk 1]
```diff
@@ -48,10 +53,17 @@ export class QuizEditionComponent implements OnInit, AfterViewInit {
...
+        mergeMap((questionId) =>
+          this.quizService.addQuestion(this.quiz.id, questionId)
+        ),
+        map((quiz) => quiz.data)
+      )
       .subscribe((quiz) => {
-        this.quiz = quiz.data;
+        this.quiz = quiz;
         this.dataSource = new MatTableDataSource<Question>(this.quiz.questions);
         this.dataSource.paginator = this.paginator;
```
Comment (ArnaudDumanois) [inline apps/front-end/back-office/src/app/components/quiz-edition/quiz-edition.component.ts:68]: what is a paginator ?

**Review by ArnaudDumanois — APPROVED** [review]
lgtm

**Comment by Apoorva64** [comment]
# ☁️ Nx Cloud Report

CI is running/has finished running commands for commit bfddded2482f38949e4023e3e40a3cd40a674a10. As they complete they will appear below. Click to see the status, the terminal output, and the build insights.

[📂 See all runs for this branch](https://cloud.nx.app/branch?workspaceId=640ee8fb56da4d6500e89717&branchName=302,281-fix-quizservice)

---


<details>
<summary>✅ Successfully ran 1 target </summary>

- [`nx serve backend-api`](https://cloud.nx.app/runs/VFnk5iIUDV)
</details>

---

Sent with 💌 from [NxCloud.](https://nx.app)

File: apps/front-end/back-office/src/app/components/quiz-edition/quiz-edition.component.ts (hunk 1 above)
Comment (Guekka) [reply apps/front-end/back-office/src/app/components/quiz-edition/quiz-edition.component.ts:68]: It's this thing

**Pull request merged**